src/EnergyPlus/SetPointManager.cc

```diff
@@ -3912,11 +3916,17 @@ namespace SetPointManager {
 
 			for ( SetPtMgrNum = 1; SetPtMgrNum <= NumOutAirSetPtMgrs; ++SetPtMgrNum ) {
 				for ( CtrlNodeIndex = 1; CtrlNodeIndex <= OutAirSetPtMgr( SetPtMgrNum ).NumCtrlNodes; ++CtrlNodeIndex ) {
+					OutAirSetPtMgr( SetPtMgrNum ).calculate();
 					NodeNum = OutAirSetPtMgr( SetPtMgrNum ).CtrlNodes( CtrlNodeIndex ); // Get the node number
 					if ( OutAirSetPtMgr( SetPtMgrNum ).CtrlTypeMode == iCtrlVarType_Temp ) {
-						// Call the CALC routine, with an optional argument to only set
-						// the initialization NODE(:)% setpoint, and not the OutAirSetPtMgr(:)%SetPt
-						OutAirSetPtMgr( SetPtMgrNum ).calculate( NodeNum, true );
+						// set to oudoor air temp reset value
```

Comment:
The "%SetPt" is placed on either node variable Node.TempSetPoint, Node.TempSetPointLo, or Node.TempSetPointHi depending on the control variable type selected.